clang/include/clang/Basic/AttrDocs.td

```diff
@@ -7552,6 +7552,27 @@ attribute requires a string literal argument to identify the handle being releas
   }];
 }
 
+def zOSExportDocs : Documentation {
+  let Category = DocCatFunction;
+  let Content = [{
+Use the ``_Export`` keyword on a function or external variable to declare
+that it is to be exported (made available to other modules). You must define
+the object name in the same translation unit in which you use the ``_Export``
```

Comment:
This documentation should also mention the pragma spelling and document the whole thing.  Also, do we diagnose that `anothony` isn't defined?